deployment/Dockerfile.go2

```
@@ -0,0 +1,26 @@
+    ros-${ROS_DISTRO}-foxglove-bridge \
+    ros-${ROS_DISTRO}-isaac-ros-apriltag \
+    ros-${ROS_DISTRO}-rosidl-generator-dds-idl \
+    ros-${ROS_DISTRO}-image-transport \
+    ros-${ROS_DISTRO}-compressed-image-transport \
+    supervisor
+
+RUN pip --no-input uninstall -y setuptools_scm torch
+RUN pip --no-input install torch torchvision torchaudio --index-url https://pypi.jetson-ai-lab.dev/jp6/cu126/+simple/
+
+
+ADD go2.requirements.txt .
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)